app/src/main/res/layout/fragment_media_detail.xml

```diff
@@ -40,257 +39,273 @@
 
             <!-- Placeholder. Height gets set at runtime based on container size; the initial value is a hack to keep
                  the detail info offscreen until it's placed properly. May be a better way to do this. -->
-            <com.facebook.drawee.view.SimpleDraweeView
-                android:id="@+id/mediaDetailImageView"
+
+            <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="@dimen/dimen_250"
-                app:actualImageScaleType="none" />
+                android:orientation="vertical"
+                android:background="@android:color/transparent"
+                android:id="@+id/mediaDetailImageViewSpacer"
+                />
 
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
-                android:background="?attr/fragmentCategorisationBackground"
-                android:orientation="vertical"
-                android:padding="@dimen/standard_gap">
+                android:background="?attr/mainBackground"
+                android:orientation="vertical">
 
                 <LinearLayout
```

Comment:
So you have this `<Linear><Text>Text></>` pattern repeated all over the place. There are 2 options that will give us actual control over the rendering here.

1. Use 3 styles, 1 for each component. Right click>refactor>extract>style
1. Create a custom [View Component](https://developer.android.com/guide/topics/ui/custom-components) so instead of declaring 3 view elements you declare 1

Reply:
@kbhardwaj123 did you see this comment?

Reply:
@kbhardwaj123 I would like to see this change implemented

Reply:
Actually the conversation thread is so lang that this slipped from being observed, I will see to this

Reply:
@macgills after inspection there are following patterns being used:
 - `<LinearLayout><text><text></>` ...(2 times)
 - `<LinearLayout><text><commons.ui.widget.HtmlTextView></>`
 - `<LinearLayout><text><commons.ui.widget.CompatTextView></>` ...(2 times)
 - `<LinearLayout><text><LinearLayout></>`

Is it fine if i extract styles for these

Reply:
Extract the styles for speed's sake I'd say

Reply:
@macgills done :), kindly see if something else needs to be improved